tests/tests/test-author-mapping.ts

```diff
@@ -117,10 +116,14 @@ describeDevMoonbeam("Author Mapping - Fail without deposit", (context) => {
       );
     }
     await context.createBlock();
+  });
+
+  it("should not add the association", async function () {
     expect(await getMappingInfo(context, bobAuthorId)).to.eq(null);
   });
 
-  it("should check events for failure", async function () {
+  // TODO: Fix this test as there is no failed extrinsic in the block
+  it.skip("should check events for failure", async function () {
```

Comment:
:eyes:  What was this one before?

Reply:
The other failure assertions in this test still seem to be working, e.g. we see:
```
      expect(e.message.toString()).to.eq(
        "1010: Invalid Transaction: Inability to pay some fees , e.g. account balance too low"
      );
```